packages/nc-gui/composables/useInfiniteData.ts

```diff
@@ -222,9 +235,9 @@ export function useInfiniteData(args: {
             },
           )
 
-      const data = formatData(response.list, response.pageInfo)
+      const data = formatData(response.list, response.pageInfo, params)
 
-      if (response.pageInfo.totalRows) {
+      if (response.pageInfo?.totalRows) {
```

Comment:
What if totalRows is `0` ? Do we need to reset ?

Reply:
fixed